script/make_utils/fix_omp_issues_for_intel_mac.sh

```diff
@@ -10,7 +10,7 @@ MACHINE=$(uname -m)
 PYTHON_VERSION=$(python --version | cut -d' ' -f2 | cut -d'.' -f1,2)
 DO_REGENERATE=0
 
-if [ "$UNAME" == "Darwin" ] && [ "$MACHINE" != "arm64" ]
```

Comment:
I don't know how it works for you, @fd0r @RomanBredehoft 
Could you tell me?

Reply:
I'm getting an error when running `make sybc_env`

Reply:
@RomanBredehoft : do you still get an error and which error please